# unique id of landmark
uint64 landmark_id

# desriptor corresponding to this landmark measurement
float32[] descriptor

# pixel coordinates where landmark was detected
float32 pixel_u
float32 pixel_v